crates/languages/src/python.rs

```diff
@@ -476,6 +492,21 @@ impl ContextProvider for PythonContextProvider {
     }
 }
 
+fn pytest_extra_args(location: Option<&Arc<dyn language::File>>, cx: &App) -> Vec<String> {
+    const PYTEST_ARGS: &str = "PYTEST_ARGS";
+    let setting = language_settings(Some(LanguageName::new("Python")), location, cx);
+    let args = setting.tasks.variables.get(PYTEST_ARGS);
+    if args.is_none() {
+        return vec![];
+    }
+    let r = shell_words::split(args.unwrap());
```

Comment:
Ugh, it'd be great to not have to do that here (our schema should ideally just be `Either<String, Array<String>>`.